models/UserSearch.php

```diff
@@ -93,8 +97,13 @@ public function search($params)
             return $dataProvider;
         }
 
-        $model = $query->modelClass;
-        $table_name = $model::tableName();
+        if (\Yii::$app->authManager instanceof yii\rbac\DbManager && $this->auth_item) {
```

Comment:
It's better to do the fast check first:
````
 $this->auth_item && \Yii::$app->authManager instanceof yii\rbac\DbManager
````
It's not too relevant, just a minor thing.